deploy/docker/Dockerfile

```diff
@@ -13,8 +14,12 @@ WORKDIR /app
 # Copy the current directory contents into the container at /app
 COPY . /app
 
+# Fetch NuCypher
+RUN git clone https://github.com/nucypher/nucypher
```

Comment:
Since we want to control which versions are out in the wild - can we specify a tagged version to install?  I know there is not one now, but there will be shortly.

Reply:
We can certainly do that, what would you recommend?

Reply:
I think there is a clone flag for this:

`git clone --branch 0.1.0a0 https://github.com/nucypher/nucypher`

Reply:
Then we can hookup to `bumpversion` to alter the string ~ Cool!